src/APICalls.js

```diff
@@ -0,0 +1,40 @@
+
+const fetchData = () => {
+    fetchUserData();
+    fetchHydrationData();
+    fetchSleepData();
+    fetchActivityData();
+}
+
+const fetchUserData = () => {
+    fetch("http://localhost:3001/api/v1/users")
+    .then(response => response.json())
+    .then(userData => console.log(userData))
+    // function that would change dom display
+    .catch(err => console.log(err.message))
+}
+
+const fetchHydrationData = () => {
+    fetch("http://localhost:3001/api/v1/hydration")
+    .then(response => response.json())
+    .then(hydrationData => console.log(hydrationData))
+    // function that would change dom display
+    .catch(err => console.log(err.message))
+}
```

Comment:
Are we sure this is the right place for DOM manipulation? Or should we keep DOM manipulation to scripts.js? Just reviewing David's solution to today's practice problems - it looks like he's keeping DOM manipulation separate from his API calls.
https://github.com/turingschool-examples/fe2-fetch-practice/blob/possible-solution/client/index.js